google-cloud-build/ci-scripts/scripts/test.sh

```diff
@@ -3,7 +3,7 @@ set -o errexit
 set -o nounset
 FLAVOR=$1
 LOG_MESSAGE=$(git log --oneline -1)
-if [[ "$LOG_MESSAGE" =~ "[skip tests]" ]]
+if [[ "$LOG_MESSAGE" =~ [skip\ tests] ]]
```

Comment:
Reason
^-- SC2076: Don't quote rhs of =~, it'll match literally rather than as a regex.

Reply:
:+1: